src/common/darktable.h

```diff
@@ -173,9 +173,16 @@ typedef enum dt_debug_thread_t
   DT_DEBUG_CAMERA_SUPPORT = 1 << 16,
 } dt_debug_thread_t;
 
+typedef struct dt_codepath_t
+{
+  unsigned int SSE2 : 1;
+  unsigned int _no_intrinsics : 1;
```

Comment:
what's the reasoning of this somewhat inconsistent naming scheme with the leading underscore?